openlibrary/plugins/openlibrary/js/scroll.js

```diff
@@ -0,0 +1,17 @@
+$(window).scroll(function() {
+    var height = $(window).scrollTop();
+    if (height > 200) {
+        $('#back2Top').fadeIn();
+    } else {
+        $('#back2Top').fadeOut();
+    }
+});
+$(document).ready(function() {
```

Comment:
You can use $( function () { here and it will be work exactly the same (Also easier to read!)